my-app/src/containers/messagePage.js

```diff
@@ -49,4 +49,4 @@ class App extends Component {
   }
 }
 
-export default App;
+export default MessagePage;
```

Comment:
1. line use `const` not `let`, you are defining an array it stays an array, you dont reassign it so it can be a `const` you just push into the array - that is not the same as reassigning.

2. line 26 OK copy and pasting? use your knowledge of looping to create your dummy content